web/src/components/common.js

```diff
@@ -12,6 +12,24 @@ function change_url(x, name=null) {
   return <a href={x.url} target="_blank" rel="noopener noreferrer">{name}</a>;
 }
 
+function filter_url(url, field) {
```

Comment:
You should use this: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get

Reply:
ok done